fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -149,14 +149,22 @@ def parse_line(line, hide_output)
     end
 
     def additional_upload_parameters
-      # Workaround because the traditional transporter broke on 1st March 2018
-      # More information https://github.com/fastlane/fastlane/issues/11958
-      # As there was no communication from Apple, we don't know if this is a temporary
-      # server outage, or something they changed without giving a heads-up
-      if ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"].to_s.length == 0
-        ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"] = "-t DAV,Signiant"
+      # As Apple recommends in Transporter User Guide we shouldn't specify the -t transport parameter
+      # and instead allow Transporter to use automatic transport discovery
+      # to determine the best transport mode for packages.
+      # It became crucial after WWDC 2020 as it leaded to "Broken pipe (Write failed)" exception
+      # More information https://github.com/fastlane/fastlane/issues/16749
+      env_deliver_additional_params = ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"]
+      if env_deliver_additional_params.nil?
```

Comment:
```suggestion
      if env_deliver_additional_params.to_s.empty?
```

One last thought! Should we change it to ☝️ so it checks either `nil` or `””`(empty string)? Not checking for empty string might break for users that are explicitly setting it to empty string 🤔 

Sorry about not catching this sooner!

Reply:
I just added a commit with this change!

Reply:
Thanks, this is a good idea. I will also add test for blank space.

Reply:
@joshdholtz I have a question. Currently, for users who set this to one space, the script will raise an error, because it isn't a valid transport parameter. I'm wondering should we live it as it is, or treat one space as if the ENV variable isn't set? Doing that the script won't raise and error for current users. What do you think?

Reply:
Ah! Good point. I’ll create a follow PR to strip it before checking if empty 🙂

Reply:
If you'd like I can do it together with tests that validate it.

Reply:
Ah, don’t worry about it! I can handle it but I appreciate the offer ❤